### ISSUE TITLE
[Task]: Config file hyperparameters

### ISSUE THREAD

**Issue by Samyarrahimi** [openvinotoolkit/anomalib#2192]
### What is the motivation for this task?

For some (and perhaps most) of the config files for models, certain options are not implemented. For example, in the draem.yaml config file, you cannot set the learning rate. As the dataset used may be custom, the preset hyperparameters might not result in optimal solutions. Is there a specific reason that some hyperparameters can't be set in some model config files? Or am I missing something here? The learning rate is just an example; this idea can be generalized to other hyperparameters and other available models in Anomalib.

### Describe the solution you'd like

I would like to be able to set different hyperparameters in the config file for all of the available models.
